.github/workflows/release_check.yml

```diff
@@ -0,0 +1,37 @@
+name: New Release check
+
+on:
+  # Triggers the workflow every day
+  schedule:
+    - cron: '0 0 * * *'
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  release_check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      
```

Comment:
Please add a separate step for "installing" python, e.g.:
```
- uses: actions/setup-python@v2
  with:
    python-version: '3.x' # Version range or exact version of a Python version to use, using SemVer's version range syntax
```
Also installing dependencies should work as follows:
```
    - name: Install dependencies
      run: |
        python -m pip install --upgrade pip
        pip install -r requirements.txt
```